feature_engine/selection/drop_features.py

```diff
@@ -52,12 +52,15 @@ class DropFeatures(BaseSelector):
 
     def __init__(self, features_to_drop: List[Union[str, int]]):
 
-        if not isinstance(features_to_drop, list) or len(features_to_drop) == 0:
+        if not isinstance(features_to_drop, (str, list)) or len(features_to_drop) == 0:
             raise ValueError(
-                "features_to_drop should be a list with the name of the variables"
+                "features_to_drop should be a list with the name of the variables "
                 "you wish to drop from the dataframe."
```

Comment:
since we are working on this class... can we add f"Got {features_to_drop} instead." to the error message?

Thank you!